doc/source/contributor/environments/ci-tenks.rst

```diff
@@ -0,0 +1,76 @@
+==========
+ci-tenks
+==========
+
+The ``ci-tenks`` Kayobe environment is used to test Seed services.
+It is is currently a work in
+progress.
+
+The environment is deployed using `automated-deployment.sh`. It bootstraps
+localhost as a hypervisor for a seed and one controller instance. The Seed
+provisions controller with Bifrost.
```

Comment:
You're capitalising seed and controller in some places, and not others. I'd suggest you just drop the capitalisation here and in the readme.
```suggestion
localhost as a hypervisor for a seed and one controller instance. The seed
provisions controller with Bifrost.
```